accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java

```diff
@@ -480,7 +480,8 @@ public void removeRemote(String remoteId) {
     LOG.i(LOG_TAG, "Remove remote with ID: ", remoteId);
     addLogEvent(ClientLog.LOG_ACTION_REMOVE_REMOTE, ClientLog.LOG_VARIATION_ATTEMPT);
     Subscriber sub = mSubscribers.get(remoteId);
-    mSubscribers.remove(sub);
+    mSubscribers.remove(remoteId);
+
```

Comment:
@marinaserranomontes  we are trying to remove Subscriber by Object instead of a key. This could be one of the issues.